### ISSUE TITLE
How to set up sales price for a product?

### ISSUE THREAD

**Issue by warpech** [handsoncode-apps/Basket#24]
In Products app `develop` branch, the product `Price` field is removed. We will not implement a naive price in that app anymore.

Because of that, Basket app sees 0 as the price for products. 

How to fix this. We may do one of the following:

1. Add Price field temporarily to the Barcodes app
2. Add Price field to a new page in the Basket app called "Price admin"
3. Hard code the Price in the Basket app

I would vote for 2. @aloscha WDYT?

**Comment by aloscha** [comment]
Yeah option two is cool, but it should present all products with modified price field - is it correct?

**Comment by warpech** [comment]
Yeah it could be just a single table with two columns: Product name and Price.
